### PR TITLE
Event filtering with use of uitablefilter

### LINKED ISSUE #12: Main page - event index creation
Main page should display the list of all upcoming events

### LINKED ISSUE #22: Event sorting and searching
Users should be able to sort events by name / game / city / date.

Users should be able to search for events by name / game / city / date.

### PR DESCRIPTION
**Pull request by KamilFilip**
closes #22 
closes #12

### CODE REVIEW

File: app/assets/javascripts/jquery.uitablefilter.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    matches(elem) : noMatch(elem);
+  });
+
+  last_phrase = phrase;
+  if( ifHidden && new_hidden ) ifHidden();
+  return jq;
+};
+
+jQuery.uiTableFilter.last_phrase = ""
+jQuery.uiTableFilter.has_words = function( str, words, caseSensitive )
```
Comment (Snieezy) [inline app/assets/javascripts/jquery.uitablefilter.js]: you should be consistent on using { }, in previous lines you used:
function {
}
convention

File: app/assets/javascripts/jquery.uitablefilter.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  if( this.last_phrase === phrase ) return false;
+
+  var phrase_length = phrase.length;
+  var words = phrase.toLowerCase().split(" ");
+
+  var matches = function(elem) { elem.show() }
+  var noMatch = function(elem) { elem.hide(); new_hidden = true }
+  var getText = function(elem) { return elem.text() }
+
+  if( column ) {
```
Comment (Snieezy) [inline app/assets/javascripts/jquery.uitablefilter.js]: you should decide if you put space before { or not (see line no. 1)

**Comment by Snieezy** [comment]
+1

**Comment by gotar** [comment]
ok you can merge

**Pull request merged**